snapshot-labs/sx-monorepo
# refactor: move the proposal start and end time to Editor

### Summary



Toward https://github.com/snapshot-labs/workflow/issues/282

This PR refactors where the proposal start/end time is being set.

Previously, this was set in networks/offchain/action.ts, since the start and end time are set at the space level.

With the upcoming feature, these values will be editable by the proposer, and should be set via the Editor.

### How to test

1. Go to the Editor
2. It should work as before, proposal start and end time should follow the voting period and delay set at the space level
